extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -358,6 +362,14 @@ class QueueProcessor extends EventEmitter {
         }
     }
 
+    _setupClientsManager() {
+        if (this.destConfig.auth.type !== libConstants.authTypeAssumeRole) {
```

Comment:
Maybe a comment here explaining why it is limited to assume role type would be great